packages/react-component-library/src/components/TabSet/TabSet.test.tsx

```diff
@@ -118,12 +131,17 @@ describe('TabSet', () => {
 
       const tabs = wrapper.getByTestId('tabs')
       tabs.scrollTo = () => {}
-      scrollToSpy = jest.spyOn(tabs, 'scrollTo').mockImplementation(x => {
-        Object.defineProperty(tabs, 'scrollLeft', {
-          configurable: true,
-          value: x,
+
+      scrollToSpy = jest
+        .spyOn(tabs, 'scrollTo')
+        // @ts-ignore
```

Comment:
What did it get upset about here?

Reply:
The destructuring of the `scrollTo` parameter.